### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by teamotitan**
Previously, for commands that uses index, if the number exceeds the max_int, a different error message will be shown instead, which is inconsistent. Due to language constraints, this happens

Let's
*Put in constraints in the UG to inform users that command will behave weirdly for values after max_int

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -188,6 +188,8 @@ Edits an existing student in the address book.
 * At least one of the optional fields must be provided. You may refer to
 [Constraints of Add command](#constraints) for acceptable values of each field.
 * Existing values will be updated to the input values.
+* Command will behave weirdly for values after 2147483647.
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:191]: Can maybe either say that command will behave weirdly for index values after the given value and group together with the other stuff about the index or just put this into the notes about command format.

**Review by Soumyadip-cmd — COMMENTED** [review]
Some considerations. Please change all instances to conform to whichever chosen

File: docs/UserGuide.md (hunk 1 above)
Comment (teamotitan) [reply docs/UserGuide.md]: Okays got it ty

File: docs/UserGuide.md [hunk 2]
```diff
@@ -348,6 +381,7 @@ Settles the amount owed by the student and adds it to the paid amount. To record
 * The **INDEX** refers to the index number shown in the displayed student list.
 * The **INDEX** **must be a positive integer** 1, 2, 3, …​
 * **AMOUNT** must be a positive value and must not be more than **OWED_AMOUNT**.
```
Comment (Feng1231) [inline docs/UserGuide.md:383]: For this one right `* **AMOUNT** must be a positive value and must not be more than **OWED_AMOUNT**.`, can you rephrase it to

`* **AMOUNT** must be a positive value and must not be more than the current amount owed by the student`

**Review by Feng1231 — APPROVED** [review]
lgtm

File: docs/UserGuide.md [hunk 3]
```diff
@@ -301,6 +304,35 @@ Finds students whose names contain any of the given keywords *and* their tuition
...
+  
+**Output:**
+![payResult.png](images/payResult.png)
+
+<box type="important" header="##### Constraints">
+
+* The **INDEX** refers to the index number shown in the displayed student list.
+* The **INDEX must be a positive integer** 1, 2, 3, …​
+* **HOURS_PAID** should be a positive multiple of 0.5, i.e. 0.5, 1.0, 1.5, etc
+* Command will behave weirdly if **INDEX** is larger than 2147483647.
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:324]: Maybe can swap the order of these 2 to group notes related to index together.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -323,6 +355,7 @@ When he/she pays the outstanding fees at a later time, you should use the [`sett
 * The **INDEX** refers to the index number shown in the displayed student list.
 * The **INDEX must be a positive integer** 1, 2, 3, …​
 * **HOURS_OWED** must be a positive multiple of 0.5, i.e. 0.5, 1.0, 1.5, etc
+* Command will behave weirdly if **INDEX** is larger than 2147483647.
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:358]: Likewise

File: docs/UserGuide.md [hunk 5]
```diff
@@ -353,7 +386,8 @@ Settles the amount owed by the student and adds it to the paid amount. To record
 
 * The **INDEX** refers to the index number shown in the displayed student list.
 * The **INDEX** **must be a positive integer** 1, 2, 3, …​
-* **AMOUNT** must be a positive value and must not be more than **OWED_AMOUNT**.
+* **AMOUNT** must be a positive value and must not be more than the current amount owed by the student.
+* Command will behave weirdly if **INDEX** is larger than 2147483647.
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:390]: Likewise

File: docs/UserGuide.md [hunk 6]
```diff
@@ -221,6 +221,8 @@ Edits an existing student in UGTeach.
 * At least one of the optional fields must be provided. You may refer to
 [Constraints of Add command](#constraints) for acceptable values of each field.
 * Existing values will be updated to the input values.
+* Command will behave weirdly if **INDEX** is larger than 2147483647.
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:224]: Likewise

**Review by Soumyadip-cmd — COMMENTED** [review]
Some suggestions

**Review by Soumyadip-cmd — APPROVED** [review]
